common/djangoapps/student/migrations/0025_auto_20191101_1846.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.25 on 2019-11-01 18:46
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from experiments.models import ExperimentData
+from openedx.features.course_duration_limits.config import EXPERIMENT_DATA_HOLDBACK_KEY, EXPERIMENT_ID
+from student.models import CourseEnrollment, FBEEnrollmentExclusion
+
+
+def populate_fbeenrollmentexclusion(apps, schema_editor):
+	for holdback_entry in ExperimentData.objects.filter(experiment_id=EXPERIMENT_ID, key=EXPERIMENT_DATA_HOLDBACK_KEY):
+	    if holdback_entry.value == 'True':
+	        for enrollment in CourseEnrollment.objects.filter(user=holdback_entry.user):
+	            FBEEnrollmentExclusion.objects.create(enrollment=enrollment)
```

Comment:
I think you can use `https://docs.djangoproject.com/en/1.11/ref/models/querysets/#bulk-create` to reduce the number of writes to the database.

Reply:
is the suggestion to insert all the records at once?  
i'm not sure how many records it would be but maybe 100k+? would that be a performance problem too?
should i use batch size and if so what batch size do you think would be performant?